app/components/com_fmns/site/assets/css/fmns.css

```diff
@@ -75,8 +75,8 @@ body {
 }
 
 .header .bcg-fmn {
-  background: url("/app/components/com_fmns/site/assets/img/art-board-network.jpg");
-  background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.7) 0%, rgba(0, 0, 0, 0.7) 100%), url("/app/components/com_fmns/site/assets/img/art-board-network.jpg");
+  background: url("/app/site/media/images/art-board-network.jpg");
```

Comment:
Just curious - better to have images in media gallery?

Reply:
If better in the media gallery (I'm fine either way), do you mind performing a `git rm` on the images in the component?

Reply:
I'm going to say to leave them in the component. I'll update the path in the css. With the exception of a couple of images, most of these won't be global.